build.gradle

```diff
@@ -61,16 +69,46 @@ repositories {
     mavenCentral()
 }
 
+// Write the plugin's classpath to a file to share with the tests
+task createClasspathManifest {
+  def outputDir = file("$buildDir/$name")
+
+  inputs.files sourceSets.main.runtimeClasspath
+  outputs.dir outputDir
+
+  doLast {
+    outputDir.mkdirs()
+    file("$outputDir/plugin-classpath.txt").text = sourceSets.main.runtimeClasspath.join("\n")
+  }
+}
+
+tasks.test.dependsOn tasks.createClasspathManifest
+if (findProject(':vendor') != null) {
```

Comment:
It's not that trivial and trust me I would love if it could be. There is 2 problems:
 1- The install task act a bit differently because of the project include manipulation happening in settings.gradle. I have to check the availability of the `:vendor` project as during the install task, the project is not include which would crash the install process.
 2- The native use the model and the java use the legacy TaskContainer. This cause problem when trying to bridge task between both world. The only way I found was through the ModelRegistry. The design doc trying to solve this issue is https://github.com/gradle/gradle/blob/master/design-docs/task-bridging.md. Some discussion happen about this can be found here: https://groups.google.com/d/topic/gradle-dev/y9WmHyzCX-s/discussion

I'm hoping to find a better way. As soon as I find one this place will be the first to get this fix.